cypress/cypress/e2e/04_submariner_addon.cy.js

```diff
@@ -0,0 +1,31 @@
+/** *****************************************************************************
+ * Licensed Materials - Property of Red Hat, Inc.
+ * Copyright (c) 2023 Red Hat, Inc.
+ ****************************************************************************** */
+
+/// <reference types="cypress" />
+
+import { clusterSetMethods } from '../views/clusterset/clusterset'
+
+describe('submariner - Deployment validation', {
```

Comment:
This test is not a deployment validation.
Please, change it to:
`submariner - validate submariner addon tab`

Reply:
done